src/main/resources/public/raml/resourceTypes/rules.raml

```diff
@@ -1,40 +1,6 @@
-description: This class is used for anything rules related. It's currently used to test rules on Eiffel events
-             and to fetch the active rules file content. Test rules must be enabled in Eiffel Intelligence
-             for the /rule-check endpoints to work.
 get:
```

Comment:
description for rules would be nice.

Reply:
Since "get" is the only thing you will be doing here and it already has a description. I did not add any "top level description" since it would say exactly the same thing as the existing description in this case.